src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Analysis.java

```diff
@@ -122,7 +134,12 @@
     @ManyToMany
     private List<File> files;
 
-    public AccessionVersionEntityId getId() {
+    public AccessionVersionId getAccessionVersionId() {
+        return accessionVersionId;
+    }
+
+    @Override
+    public Long getId() {
```

Comment:
It is a minor detail, but maybe it would make more sense to list `getId` first in all the models, given that is the main identifier instead of accession+version.